tutorials/operators/01_operator_flow.ipynb

```diff
@@ -0,0 +1,2251 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 50,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "SummedOp([\n",
+      "  -1.0523732 * II,\n",
+      "  0.39793742 * IZ,\n",
+      "  -0.3979374 * ZI,\n",
+      "  -0.0112801 * ZZ,\n",
+      "  0.18093119 * XX\n",
+      "])\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(two_qubit_H2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we multiply the Hamiltonian by a `Parameter`. This `Parameter` is stored in the `coeff` property of the `SummedOp`. Calling `exp_i()` on the result wraps it in `EvolvedOp`, representing exponentiation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "e^(-i*1.0*θ * SummedOp([\n",
+      "  -1.0523732 * II,\n",
+      "  0.39793742 * IZ,\n",
+      "  -0.3979374 * ZI,\n",
+      "  -0.0112801 * ZZ,\n",
+      "  0.18093119 * XX\n",
+      "]))\n",
+      "EvolvedOp(SummedOp([PauliOp(Pauli(z=[False, False], x=[False, False]), coeff=-1.0523732), PauliOp(Pauli(z=[True, False], x=[False, False]), coeff=0.39793742), PauliOp(Pauli(z=[False, True], x=[False, False]), coeff=-0.3979374), PauliOp(Pauli(z=[True, True], x=[False, False]), coeff=-0.0112801), PauliOp(Pauli(z=[False, False], x=[True, True]), coeff=0.18093119)], coeff=1.0*θ, abelian=False), coeff=1.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "evo_time = Parameter('θ')\n",
+    "evolution_op = (evo_time*two_qubit_H2).exp_i()\n",
+    "print(evolution_op) # Note, EvolvedOps print as exponentiations\n",
+    "print(repr(evolution_op))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We construct `h2_measurement`, which represents `two_qubit_H2` as an observable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OperatorMeasurement(SummedOp([\n",
+      "  -1.0523732 * II,\n",
+      "  0.39793742 * IZ,\n",
+      "  -0.3979374 * ZI,\n",
+      "  -0.0112801 * ZZ,\n",
+      "  0.18093119 * XX\n",
+      "]))\n"
+     ]
+    }
+   ],
+   "source": [
+    "h2_measurement = StateFn(two_qubit_H2).adjoint()\n",
+    "print(h2_measurement)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We construct a Bell state $|\\Phi_+\\rangle$ via $\\text{CX} (H\\otimes I) |00\\rangle$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CircuitStateFn(\n",
+      "               \n",
+      "q_0: ───────■──\n",
```

Comment:
Hadamard gate is for the qubit 0. I think `bell = CX @ (I^H) @ Zero` is correct

Reply:
Yes, you are correct. The circuit is clearly wrong.